phpstan.neon

```diff
@@ -0,0 +1,4 @@
+parameters:
+    level: 8
+    paths:
+        - src/
```

Comment:
iterable type can be ignored, see other modules

Reply:
Added checkMissingIterableValueType: false to phpstan.neon